base/loading.jl

```diff
@@ -724,12 +726,14 @@ function stale_cachefile(modpath::String, cachefile::String)
         provides_concrete && return false # this is the file we want
 
         # now check if this file is fresh relative to its source files
-        if files[1][1] != modpath
+        if normpath(files[1][1]) != normpath(modpath)
```

Comment:
When we relocate JI files, we edit the paths that are stored in the JI files. For e.g.
If the original path was `/private/tmp/Applications/JuliaPro-0.5.0.1/Contents/Resources/pkgs-0.5.0.1/v0.5/ZMQ/src/ZMQ.jl` it gets modified to `/////////////Applications/JuliaPro-0.5.0.1/Contents/Resources/pkgs-0.5.0.1/v0.5/ZMQ/src/ZMQ.jl`. Although the above two paths are the same, string comparison would fail. Hence the change.